packages/version/src/version-command.ts

```diff
@@ -658,15 +659,27 @@ export class VersionCommand extends Command<VersionCommandOption> {
             break;
           case 'npm':
           default:
-            // with npm, we need to do update the lock file in 2 steps
-            // 1. use shrinkwrap will delete current lock file and create new "npm-shrinkwrap.json" but will avoid npm retrieving package version info from registry
-            lockFilename = 'package-lock.json';
-            this.logger.verbose(`lock`, `updating lock file via "npm shrinkwrap --package-lock-only"`);
-            await exec('npm', ['shrinkwrap', '--package-lock-only'], { cwd: this.project.manifest.location });
-
-            // 2. rename "npm-shrinkwrap.json" back to "package-lock.json"
-            this.logger.verbose(`lock`, `renaming "npm-shrinkwrap.json" file back to "package-lock.json"`);
-            renameSync('npm-shrinkwrap.json', 'package-lock.json');
+            const localNpmVersion = execSync('npm', ['--version']);
+            this.logger.silly(`npm`, `current local npm version is "${localNpmVersion}"`);
+
+            // for npm version >=8.5.0 we can call "npm install --package-lock-only"
+            // when lower than we call "npm install --package-lock-only" and rename "npm-shrinkwrap.json" back to "package-lock.json"
+            if (semver.gte(localNpmVersion, '8.5.0')) {
+              this.logger.verbose(`lock`, `updating lock file via "npm install --package-lock-only"`);
+              await exec('npm', ['install', '--package-lock-only'], { cwd: this.project.manifest.location });
+            } else {
+              // with npm, we need to do update the lock file in 2 steps
+              // 1. use shrinkwrap will delete current lock file and create new "npm-shrinkwrap.json" but will avoid npm retrieving package version info from registry
+              lockFilename = 'package-lock.json';
+              this.logger.verbose(`lock`, `updating lock file via "npm shrinkwrap --package-lock-only".`);
+              this.logger.warn(`npm`, `Your npm version is lower than 8.5.0, we strongly suggest to upgrade your npm client to avoid the use of "npm shrinkwrap" instead of the regular "npm install --package-lock-only".`);
+              await exec('npm', ['shrinkwrap', '--package-lock-only'], { cwd: this.project.manifest.location });
```

Comment:
I'm not sure if `npm-shrinkwrap.json` is really backwards compatible with the `package-lock.json`. 8.5 is shipped with Node.js LTS. I'd show an error that users need to update to NPM >= 8.5.0 this is practical.

Reply:
Oh yes they are identical https://docs.npmjs.com/cli/v8/configuring-npm/npm-shrinkwrap-json

Reply:
I am on Node 16 and like I mentioned earlier, I was on 8.4.1 even though I'm on Node LTS, so I don't think this problem will disappear anytime soon and so the `logger.warn` does exactly that to display a warning and recommend to upgrade npm.

Reply:
I mean latest LTS. npm 8.5.0 was shipped with Node.js 16.15.0, Current LTS is 16.15.1

Reply:
sure but Node 16 LTS is only EOL on June 2024 and maybe another 6 months buffer then it's better to keep this `shrinkwrap` in place as a plan B for the foreseeable future haha. I also don't have the tendency to update very often Node myself so the plan B is a good thing. I'll consider this one resolved